instrumentation-core/spring/spring-boot-autoconfigure/README.md

```diff
@@ -0,0 +1,428 @@
+# OpenTelemetry Spring Auto-Configuration
+
+Auto-configures OpenTelemetry instrumentation for [spring-web](../spring-web-3.1/), [spring-webmvc](../spring-webmvc-3.1/), and [spring-webflux](../spring-webflux-5.0/). Leverages Spring Aspect Oriented Programming, dependency injection, and bean post-processing to trace spring applications. To include use all features listed below use the [opentelemetry-spring-starter](../starters/spring-starter/README.md).
+
+## Quickstart
+
+### Add these dependencies to your project.
+
+Replace `OPENTELEMETRY_VERSION` with the latest stable [release](https://mvnrepository.com/artifact/io.opentelemetry).
```

Comment:
I think better to link to official site, e.g. https://search.maven.org/search?q=g:io.opentelemetry

also, we aren't publishing to maven central yet from opentelemetry-java-instrumentation, so maybe include somewhere that for now you need to add our bintray maven repository: https://dl.bintray.com/open-telemetry/maven/